components/hab/src/command/pkg/delete.rs

```diff
@@ -0,0 +1,47 @@
+//! Delete a package from Builder.
+//!
+//! # Examples
+//!
+//! ```bash
+//! $ hab pkg delete acme/redis/2.0.7/2112010203120101 stable
+//! ```
+//! //! This will delete the acme package specified from Builder
+//! //! if certain conditions apply - for example, the package is not
+//! //! in the stable channel, and does not have any other packages that
+//! //! depend on it.
+//!
+//! //! Note: This command does not remove the package from disk
+
+use crate::{api_client::Client,
+            common::ui::{Status,
+                         UIWriter,
+                         UI},
+            hcore::package::PackageIdent};
+
```

Comment:
We're trying to standardize on one block for all `use` statements with blank lines so `rustfmt` sorts them for us.

Reply:
OK